pkg/server/endpoints/node/handler.go

```diff
@@ -36,6 +36,8 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+const fetchSVIDCacheSize = 100_000
```

Comment:
Can you add the units of this size const either as a suffix of the const name or as a comment?

Reply:
Added comment